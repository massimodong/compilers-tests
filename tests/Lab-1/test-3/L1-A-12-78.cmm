int UZvtRgWYm5zMk ( ) {
}
struct EsIacxbwngNlE zImUjb95JnBfr , D9zbC 

;
struct xYKK9vodkLf3J9Zq2S {
  float D55G_8cdtU0c [ 861899281 ] ;
};
int ;
struct W5c3cv7Y0FZ ;
int ;
struct L_ ;
int ;
int ;
struct z o5ko2zua9 
;
int hxRQ8K5QzAzFwuotA ( int d65U9hQhJ_Kf1jYk ) {
  int nYj4 , YfaHb = ;
  if ( KwxfOy #)   pEpkgc = HA2rdh != OSj1z - t8Ayp 
    * - QMwdbedgxGId33Q ( mbhD3Z0 , GO_gEC || Wu && u3kSJGOFG4F 
    >= Fvqqq8ebcRpf2BfsBW * wwg4RdJFy_7j [ JvMAdbZ && 
    ARpsumZ9sEYuGCcqpGL / ( UO8Sgtq4 < - ! N8F2Ba50jJ 
    [ NDIfBjN6Pqb4GqrIPs = oUAhQvYbIwHeCWv9G2Ti == pXnGiXvEz4o 
    / ( jTD3WqPLa7qRg = y8GLOn4evNYiICMdf > tLkZewu6XYoj 
    ) * ubMHZyXkKoI7sV + ! 1956480644 <= ! - iuAne1JCBi7L3XaT2 
    ] ) ] / YujU2PW99yS ( ) + ! ! ! 127307857.2118054518 
    ) . ;

'}
struct {
};
int ;
int YI7_zuSoqypbDjOI8Cdx ( int SEmUPQ int F0C ) {
}
int ;
int jTVeQ ( ) {
}
int ;
int ;
int ;
int ;
int ;
int YWJpqtQ76bMbm_ugIS ( ) {
}
int ;
int r22 ( ) {
  if ( t20Oz4JsB )   while ( LuNDmjn1hTWGqkp7 ) {
  }
else   if ( QrCj1HKYoVWTi7B5 )   return i ;


}
int ;
int ~GM ( ) {
}
int ;
int ;
int ;
int fVdI ( ) {
  dKyncv ;
  ieOoiGrurJtjCL ;
  if ( FSZwF0CjcS3t1LoUaNa )   GPA_A5OQwHW4PV1AK ;
else   jwODGCuYvFXBS6 ;

}
int ;
