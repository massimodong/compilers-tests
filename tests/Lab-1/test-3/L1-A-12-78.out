Error Type B at Line 18
Error Type A at Line 19
Error Type B at Line 27
Error Type A at Line 29
Error Type B at Line 33
Error Type A at Line 54
